backend/src/common/error/AppError.ts

```diff
@@ -8,7 +8,7 @@ export class AppError extends Error {
   public errorMessage: string;
   public userMessage: string;
 
-  constructor(errorCode: AppErrorTypeEnum) {
+  constructor(errorCode: AppErrorTypeEnum, msg: string = undefined) {
```

Comment:
Vietoj `msg: string = undefined` reiketu naudoti funkcijos optional argumentus `msg? : string`
https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters